src/sgct/readconfig.cpp

```diff
@@ -2489,5 +2501,14 @@ std::string serializeConfig(const config::Cluster& cluster) {
     return res.dump(2);
 }
 
+std::string serializeConfig(const config::Cluster& cluster,
+                            const config::GeneratorVersion& genVersion)
+{
+    nlohmann::json res;
+    res["version"] = 1;
```

Comment:
This should be:
```cpp
res["major"] = 1;
res["minor"] = 0;
```
no?

Reply:
The `res["version"] = 1;` is the original version, applying to the format of the config file rather than the version of the generator tool.